src/main/java/com/databricks/spark/avro/AvroUtils.java

```diff
@@ -24,7 +24,13 @@
 public class AvroUtils {
     /** Returns a Schema RDD for the given avro path. */
     public static JavaSchemaRDD avroFile(JavaSQLContext sqlContext, String path) {
-        AvroRelation relation = new AvroRelation(path, sqlContext.sqlContext());
+        AvroRelation relation = new AvroRelation(path, 0, sqlContext.sqlContext());
+        return sqlContext.baseRelationToSchemaRDD(relation);
```

Comment:
Maybe just use `return AvroUtils.avroFile (sqlContext, path, 0);` to avoid duplication.